pkg/network/tracer/tracer.go

```diff
@@ -812,18 +816,21 @@ func newHTTPMonitor(c *config.Config, tracer connection.Tracer, bpfTelemetry *te
 
 	monitor, err := http.NewMonitor(c, offsets, sockFDMap, bpfTelemetry)
 	if err != nil {
-		log.Errorf("could not instantiate http monitor: %s", err)
+		http.USMStartupError = fmt.Errorf("could not instantiate http monitor: %s", err)
```

Comment:
Can't all of these errors be moved into the `http` package? You don't have to export `USMStartupError` then.

Reply:
only if we move the entire function, as this is the only place that knows why USM didn't start

Reply:
I meant set `USMStartupError` inside `NewMonitor`?